coding-the-legislation/legislation_parameters.md

```diff
@@ -80,4 +80,8 @@ Example: if the minimum wage is removed from legislation in 2017, the previous e
 
 ## Usage in formulas
 
-See [this example](coding-the-legislation/10_basic_example.html#example-with-legislation-parameters).
\ No newline at end of file
+See [this example](coding-the-legislation/10_basic_example.html#example-with-legislation-parameters).
+
+## Importing from IPP tables
```

Comment:
As someone who is reading the tutorial about coding the legislation, I cannot understand this without more context.

- What is IPP ?
- Can I use IPP tables for `openfisca-canada` ?

We should clearly indicates that, in the specific case of OpenFisca-France, we have a source of parameters named IPP, which provides up parameters history, etc.